susi_python/__init__.py

```diff
@@ -1,11 +1 @@
 from .main import susi_client
-
-use_api_endpoint = susi_client.use_base_url
-query = susi_client.query
-ask = susi_client.ask
-answer_from_json = susi_client.answer_from_json
-sign_in = susi_client.sign_in
-sign_up = susi_client.sign_up
-forgot_password = susi_client.forgot_password
-get_previous_responses = susi_client.get_previous_responses
-update_location = susi_client.update_location
```

Comment:
Removing these variables isn't enough, because it will break `susi_linux` and maybe other clients. We need to search for uses of this library within the susi environment, and provide patches that remove the use of these vars. Only **after** the uses have been fixed, we can actually remove that!

Reply:
okay @norbusan , lets just keep it as wip then and I'll open a dependant issue in SUSI Linux and then we'll merge them simultaneously?

Reply:
Yes!